main.py

```diff
@@ -376,20 +373,20 @@ def emailLogForCourseID(courseID, recipients):
         return
 
     try:
-        READ_BINARY_MODE = 'rb'
-        logfile = open(getCourseLogFilePath(courseID), mode=READ_BINARY_MODE)
+        logfile = open(getCourseLogFilePath(courseID))
```

Comment:
Any reason it shouldn't use binary mode to read the file?

Reply:
Well, if I use `rb`, then when you call `.read()` on the file object, it returns a bytes string. As I noted in my PR description, passing a bytes string to `message.set_content()` throws an error. I'm just not sure whether there is any benefit to having it continue to be a bytes string; if there is, I can figure out how to make it work.

Reply:
OK.  I'd forgotten that the byte string was the result of using `rb`.